src/frontend/src/pages/Calendar/SummaryModal/SummaryComponents/DesignReviewSummaryModalAttendees.tsx

```diff
@@ -0,0 +1,77 @@
+import { Grid, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import { DesignReview, User } from 'shared';
+import { useState } from 'react';
+import { DesignReviewMemberPill } from '../../../../components/DesignReviewMemberPill';
+import { useToast } from '../../../../hooks/toasts.hooks';
+import { useCurrentUser } from '../../../../hooks/users.hooks';
+
+interface DesignReviewSummaryModalAttendeesProps {
+  designReview: DesignReview;
+}
+
+const DesignReviewSummaryModalAttendees: React.FC<DesignReviewSummaryModalAttendeesProps> = ({ designReview }) => {
+  const toast = useToast();
+  const [requiredMembers, setRequiredMembers] = useState<User[]>(designReview.requiredMembers);
+  const [optionalMembers, setOptionalMembers] = useState<User[]>(designReview.optionalMembers);
+  const currentUser = useCurrentUser();
+  const handleRemoveRequiredMember = (user: User) => {
+    if (currentUser.userId === designReview.userCreated.userId) {
+      setRequiredMembers(requiredMembers.filter((member) => member.userId !== user.userId));
+    } else {
+      toast.error('Only the creator of the Design Review can edit attendees');
+    }
+  };
+
+  const handleRemoveOptionalMember = (user: User) => {
+    if (currentUser.userId === designReview.userCreated.userId) {
+      setOptionalMembers(optionalMembers.filter((member) => member.userId !== user.userId));
+    } else {
+      toast.error('Only the creator of the Design Review can edit attendees');
+    }
+  };
+
+  return (
+    <Box marginLeft="15px">
+      <Grid container direction="row" paddingY="6px">
```

Comment:
you can remove this grid container, move the padding info to the above box, and then make the two grid containers below only grid containers and not also items. This should make the code easier to read through